tests/system/readme.md

```diff
@@ -103,3 +103,47 @@ NVDA is started with the `-c` option to specify this profile directory to be use
 Both Robot Framework and NVDA logs are captured in the `testOutput` directory in the repo root.
 NVDA logs (NVDA log, stdOut, and stdErr for each test) are under the `nvdaTestRunLogs` directory. 
 The log files are named by suite and test name.
+
+### Comparing changes to NVDA Settings
+To check for unreleased changes to the settings dialogs, one can run system tests against two copies of NVDA. 
+
+`.\runsettingsdiff.bat` can be used to compare the settings dialog by reading text and generating screenshots for comparison. 
+

```

Comment:
Please be explicit about the output locations. Screenshots will need to be compared manually with a difftool, but devs need to know where the files will go (and know they will not override each other. 

It would be easiest if they could be compared folder wise: `diffToolX currentVersion-screenshots/ oldVersion-screenshots/`